src/nti/fakestatsd/tests/test_fakestatsd.py

```diff
@@ -136,3 +153,16 @@ def test_clear(self):
 
         self.client.clear()
         assert_that(self.client, has_length(0))
+
+
+    def test_tracks_multimetrics(self):
+        packet = b'gorets:1|c\nglork:320|ms\ngaugor:333|g\nuniques:765|s'
```

Comment:
There seems to be a typing mismatch here leading to test failures on Python 3.

Reply:
That's configurable in .travis.yml. Usually it's off because, wow, that's a lot of email. And there's not usually a single email address that's suitable, many OS projects have multiple maintainers/reviewers.

Also, `tox` is a great tool to run before pushing a PR to find these kind of issues.